tools/src/test/scala/caliban/tools/SchemaWriterSpec.scala

```diff
@@ -721,6 +721,7 @@ object SchemaWriterSpec extends ZIOSpecDefault {
         |  sealed trait AllErrors extends scala.Product with scala.Serializable derives caliban.schema.Schema.SemiAuto
         |  @GQLInterface
         |  sealed trait Error extends scala.Product with scala.Serializable derives caliban.schema.Schema.SemiAuto {
```

Comment:
As a side note, in Scala 3 we probably don't need the `extends scala.Product with scala.Serializable`. We can perhaps look into removing at some other time